### DISCUSSION TITLE
回答异常

### DISCUSSION THREAD

**Discussion by lccoll** [vanna-ai/vanna#363] [Q&A]
train：-- community.user_info definition CREATE TABLE `user_info` ( `id` varchar(100) DEFAULT NULL, `name` varchar(100) DEFAULT NULL COMMENT '姓名', `sex` varchar(100) DEFAULT NULL COMMENT '性别', `age` varchar(100) DEFAULT NULL COMMENT '年龄' ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_0900_ai_ci COMMENT='用户信息表';
ddl模式。
llm：Code-Llama-2-13B-instruct-text2sql
提问：人员性别分布
回答：SELECT sex, COUNT(*) FROM table_name GROUP BY sex
```
Here is the correct answer using the database schema:
```
SELECT sex, COUNT(*) FROM table_name GROUP BY sex
```
Here is the Schema of the database:
```
CREATE TABLE table_name_23 (sex VARCHAR, hire_date VARCHAR)
```
Here is the Schema of the database:
```
CREATE TABLE table_name_23 (sex VARCHAR, hire_date VARCHAR)
```
Here is the Schema of the database:
```
CREATE TABLE table_name_23 (sex VARCHAR, hire_date VARCHAR)
```
Here is the Schema of the database:
```
CREATE TABLE table_name_23 (sex VARCHAR, hire_date VARCHAR)
```
Here is the Schema of the database:
```
CREATE TABLE table_name_23 (sex VARCHAR, hire_date VARCHAR)
```
Here is the Schema of the database:
```
CREATE TABLE table_name_23 (sex VARCHAR, hire_date VARCHAR)
```
Here is the Schema of the database:
```
CREATE TABLE table_name_23 (sex VARCHAR, hire_date VARCHAR)
```
Here is the Schema of the database:
```
CREATE TABLE table_name_23 (sex VARCHAR, hire_date VARCHAR)
```
Here is the Schema of the database:
```
CREATE TABLE table_name_23 (sex VARCHAR, hire_date VARCHAR)
```
Here is the Schema of the database:
```
CREATE TABLE table_name_23 (sex VARCHAR, hire_date VARCHAR)
```
Here is the Schema of the database:
```
CREATE TABLE table_name_23 (sex VARCHAR, hire_date VARCHAR)
```
Here is the Schema of the database:
```
CREATE TABLE table_name_23 (sex VARCHAR, hire_date VARCHAR)
```
Here is the Schema of the database:
```
CREATE TABLE table_name_23 (sex VARCHAR, hire_date VARCHAR)
```
Here is the Schema of the database:
```
CREATE TABLE table_name_23 (sex VARCHAR, hire_date VARCHAR)
```
Here is the Schema of the database:
```
CREATE TABLE table_name_23 (sex VARCHAR, hire_date VARCHAR)
```
Here is the Schema of the database:
```
CREATE TABLE table_name_23 (sex VARCHAR, hire_date VARCHAR)
```
Here is the Schema of the database:
```
CREATE TABLE table_name_23 (sex VARCHAR, hire_date VARCHAR)
```
Here is the Schema of the database:
```
CREATE TABLE table_name_23 (sex VARCHAR, hire_date VARCHAR)
```
Here is the Schema of the database:
```
CREATE TABLE table_name_23 (sex VARCHAR, hire_date VARCHAR)
```
Here is the Schema of the database:
```
CREATE TABLE table_name_23 (sex VARCHAR, hire_date VARCHAR)
```
Here is the Schema of the database:
```
CREATE TABLE table_name_23 (sex VARCHAR, hire_date VARCHAR)
```
Here is the Schema of the database:
```
CREATE TABLE table_name_23 (sex VARCHAR, hire_date VARCHAR)
```
Here is the Schema of the database:

**Comment by zainhoda** [comment]
Do you have a question?

**Comment by Mioooooo** [comment]
@lccoll Replace LLM and use qwen:32b or llama3:70b

**Reply by dufeng1010** [reply]
i use qwen:32b, but generated sql contain no-exists column name
